infra/openshift-manifests/lib.sh

```diff
@@ -45,3 +45,26 @@ EOF
 function delete_minio_endpoint_route(){
     oc delete route -n minio-operator minio-endpoint
 }
+
+function patch_knative_serving(){
+    # patch knative serving to use http instead of https in the service status url
+    oc patch knativeserving -n knative-serving knative-serving -p '{"spec":{"config":{"network":{"default-external-scheme": "http"}}}}' --type=merge
+    # wait until knative serving is ready
+    oc wait --for=condition=Ready knativeserving -n knative-serving knative-serving
+}
```

Comment:
Ideally, we do this with kustomize

Reply:
Can you make that for me? I am busy with implementing the next round of services.
I don't know about Kustomize.

Reply:
it is not a blocker comment